modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/main/java/dev/galasa/framework/k8s/controller/TestPodScheduler.java

```diff
@@ -143,7 +142,21 @@ public void run() {
                 startPod(selectedRun);
 
                 if (!queuedRuns.isEmpty()) {
-                    Thread.sleep(600); // *** Slight delay to allow Kubernetes to catch up
+                    // Slight delay to allow Kubernetes to catch up....
+                    //
+                    // Why do this ? 
+                    //
+                    // If we don't do this, then all the tests get scheduled on the same node, and the 
+                    // node will run out of memory.
+                    //
+                    // We assume that's because the usage statistics on a pod are not synchronized totally at
+                    // real-time, but have a lag in which they catch up. Hopefully this delay is greater
+                    // than the lag and when we actually schedule the next pod it gets evenly distributed over
+                    // the nodes which are available.
+                    //
+                    // This may or may not be necessary if the scheduling policies in the cluster are changed. Not sure.
+                    long launchIntervalMilliseconds = cpsFacade.getKubeLaunchIntervalMilliseconds();
```

Comment:
Cool - saw the previous hard coded delay when looking through the code for the inttests Epic and didn't like it so glad thats been improved now.